### DISCUSSION TITLE
github.community answers/solutions don't count for Galaxy Brain?

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#18308] [Other Feature Feedback, Questions, & Ideas]
Okay so cool we have [new achievements](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/). Some of which I'm proud to say I've knocked off. But it surprises me that github.community answers (solutions) don't apply for Galaxy Brain?

**Comment by p14** [comment]
Congrats by the way, over 1000 PRs merged seems wild, but your consistency shows! The community answers should still count, but I believe the answers have to be flagged by the OP, not just participation... if that makes sense.

**Reply by kingthorin** [reply]
Thanks! :grinning:

I was thinking like these: https://github.community/u/kingthorin/activity/solved (for everyone/anyone that participates in github.community discussions)

**Comment by saltbo** [comment]
Hi there! The Galaxy Brain Achievement requires the qualifying Discussions to be marked as answers by another user that's not the author.
